site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -86,6 +87,7 @@ public function uploadGradeable(): JsonResponse {
         $values['id'] = $_POST['id'];
         $values['title'] = $_POST['title'];
         $values['type'] = $_POST['type'];
+        $values['autograding_config_path'] = $_POST['autograding_config_path'] ?? FileUtils::joinPaths($this->core->getConfig()->getSubmittyInstallPath(), 'more_autograding_examples/upload_only/config');
```

Comment:
do we need any validation/security checks on the user-inputted path, or do we trust instructors & our access control enough?

Reply:
After looking through the code, the input is getting checked when the gradeable object is created, and the upload function calls the function that creates the gradeable, so the check is happening a little bit down the line, but still happening.